lib/discourse_inflector.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class DiscourseInflector < Zeitwerk::Inflector
+  def camelize(basename, abspath)
+    return basename.camelize if abspath.ends_with?("onceoff.rb")
+    return 'Jobs' if abspath.ends_with?("jobs/base.rb")
+    super
```

Comment:
I think it'll be helpful if we can add some comments in this file about why we need to carry our custom inflector here.

Reply:
This is what we were doing already in `lib/zeitwerk_config.rb` actually, I’ve created a custom inflector because that’s how you do it now, our previous code wasn’t working anymore.

Reply:
oo I see why we need this here. Basically the constants we define in the jobs folder do not follow the proper convention. Fixing this could be a pain now... perhaps we can leave a comment here on why this monkey patch is required?